qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1517A.py

```diff
@@ -1,18 +1,332 @@
+import re
 import textwrap
 from typing import Optional, Dict, Any, Union, TYPE_CHECKING
 import numpy as np
 import qcodes.utils.validators as vals
+from qcodes import InstrumentChannel, MultiParameter
```

Comment:
Please don't use imports from toplevel within qcodes. They are likely to result in circular dependencies

Reply:
Yes, I was using auto imports of pycharm and hoping it will do the proper imports. Fixed it now.